clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -517,7 +552,7 @@ class CoverageMappingBuilder {
     SourceRegionFilter Filter;
     for (const auto &FM : FileIDMapping) {
       SourceLocation ExpandedLoc = FM.second.second;
-      SourceLocation ParentLoc = getIncludeOrExpansionLoc(ExpandedLoc);
+      SourceLocation ParentLoc = getIncludeOrExpansionLoc(ExpandedLoc, false);
```

Comment:
The boolean parameter for this function is only used in this single callsite. Do we need it at all?

Reply:
I thought it was only used in one place, oops. :) The current way is fine.